index.html

```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
+                        <div class="social-icons">
+                            <ul>
+                                <li>
+                                    <a href="https://twitter.com/KissiKenneth"><img src="asset/twitter.svg" alt="twitter icon"></a>
+                                </li>
+                                <li>
+                                    <a href="https://www.linkedin.com/in/kenneth-agyare-kissi-673a01186/"><img src="asset/Linkedin icon.svg" alt="Linkedin icon"></a>
+                                </li>
+                                <li>
+                                    <a href="#"><img src="asset/M.svg" alt="M icon"></a>
+                                </li>
+                                <li>
+                                    <a href="https://github.com/over-geek"><img src="asset/github.svg" alt="github icon"></a>
+                                </li>
+                                <li>
+                                    <a href="#"><img src="asset/thumbs.svg" alt="thumbs icon"></a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div>
+                        <button class="btn">Get my resume</button>
+                    </div>
+                </div>
+                <div class="skill-list">
+                    <div class="languages">
+                        <div class="language-header">
+                            <p>Languages</p>
+                            <button>
+                                <img src="asset/chevron-down.png" alt="arrow icon">
+                            </button>
+                        </div>
+                        <div class="language-list">
+                            <div>
+                                <img src="asset/js-icon.png" alt="js icon">
+                                <p>Javascript</p>
+                            </div>
+                            <div>
+                                <img src="asset/html-icon.png" alt="html icon">
+                                <p>HTML</p>
+                            </div>
+                            <div>
+                                <img src="asset/css-icon.png" alt="css icon">
+                                <p>CSS</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="frameworks">
+                        <p>Frameworks</p>
+                        <button>
+                            <img src="asset/chevron-right.png" alt="arrow icon">
+                        </button>
+                    </div>
+                    <div class="skills">
+                        <p>Skills</p>
+                        <button>
+                            <img src="asset/chevron-right.png" alt="arrow icon">
+                        </button>
+                    </div>
+                </div>
+            </div>
```

Comment:
- [ ] Kindly make sure to wrap the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`. :wink: 
- [ ] KIndly make sure to wrap the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.